### ISSUE TITLE
Compiler warnings after clean install with Yarn 2

### ISSUE THREAD

**Issue by dennisjagut** [handsontable/react-handsontable#176]
### Description
Yarn compiles with warnings after installing and using Handsontable.
`./.yarn/cache/pikaday-npm-1.8.0-ea539e5751-2.zip/node_modules/pikaday/pikaday.js`
`Module not found: A package is trying to access another package without the second one being listed as a dependency of the first one`

I expect no compiler warnings after a clean install.

### Steps to reproduce

1. yarn dlx create-react-app hottest --template typescript
2. yarn add handsontable @handsontable/react 
3. Add `<HotTable/>` to the React App
4. yarn start

### Your environment
* React wrapper version: 3.1.2
* Handsontable version: 7.4.2
* Operating System: macOS 10.15.3
* node: v13.8.0
* yarn: 2.0.0-rc.29

**Comment by pnowak** [comment]
Hi @dennisjagut 

I can't replicate the issue.

Could you share with us your `package.json` and the rest of your code?

**Comment by dennisjagut** [comment]
Sure, I created a repository with all files and instructions in the `README.md`

https://github.com/dennisjagut/handsontable-yarn2-test

Additionally, I updated my environment in the initial post with the yarn and node version which I used

**Comment by pnowak** [comment]
Hi @dennisjagut 

Sorry for a delay.

Yes, I got the same message warning. 

I have tried to resolve this issue by using `packageExtensions` - https://yarnpkg.com/configuration/yarnrc#packageExtensions. Unfortunately without success. 
After I added `yarnrc.yml` file with 
```packageExtensions:
  "pikaday":
    dependencies:
      "moment": "*"```
I removed `node_modules`. Then I installed modules by `npm install` and the use `yarn`. At the end `yarn start`. But the problem doesn't disappear.

Maybe I missed something else 🤔
